internal/net_util/net_util.go

```diff
@@ -0,0 +1,17 @@
+package net_util
```

Comment:
I suggest the file path using `internal/utils/net.go`.
```suggestion
package utils
```

Reply:
It's done :+1: